_docs/00-getting-started.md

```diff
@@ -30,6 +36,28 @@ curl -sSL "https://github.com/facebook/infer/releases/download/v$VERSION/infer-l
 ln -s "/opt/infer-linux64-v$VERSION/bin/infer" /usr/local/bin/infer
 ```
 
+Use this to download the **Mac version** of infer to `/Applications` and optionally put
+it on the `PATH`. You should replace `0.16.0` with the [latest release version](https://github.com/facebook/infer/releases/latest):
+
+```sh
+curl -O -L "https://github.com/facebook/infer/releases/download/v0.16.0/infer-osx-v0.16.0.tar.xz"
```

Comment:
Why not copy the instructions for Linux, replacing /opt with /Applications? You could also re-use the symbolic link method to get infer in the `PATH` to avoid having to edit .bash_profile.

Reply:
I'll add the symbolic link option as well, but it has a disadvantage of creating another file in an obscure location. It's probably easy to forget to delete this file when uninstalling infer. But nevermind.

However, I can't reuse the Linux instructions because:

* The tar and curl flags used in the Linux example do not work on MacOS. I tried it, and it just started spitting out many MB of binary data to the Terminal, and I had to force quit the command.

* I consider it too complicated to combine all those commands into one line or one command. It's not clear what each part does, and it's hard to copy it as a long string. When it's broken up into separate commands, it's easier to understand what each part does. And the user can pause in between each step too.

* I consider the `"VERSION=0.XX.Y"` string substitution somewhat complicated for users who want to install the software manually. It's ok for something like an automated script, but not for manual typing. Another reason: The Terminal/shell will auto-complete filenames and paths when pressing Tab; but this won't happen if the filename contains a string substitution.